src/test/java/products/ProductTest.java

```diff
@@ -0,0 +1,38 @@
+package test.java.products;
+
+import main.java.products.DeviceType;
+import main.java.products.Product;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ProductTest {
+
+    @Test
+    public void product_withOutBrand() {
```

Comment:
Anche se è solo il nome di un test, cerca di fare attenzione alla scrittura in quanto è "without", non sono due parole